content/en/docs/tasks/security/health-check/index.md

```diff
@@ -37,7 +37,7 @@ See the [authentication policy task](/docs/tasks/security/authn-policy/) for det
 To enable health checking, redeploy Citadel:
 
 {{< text bash >}}
-$ istioctl manifest generate --set values.global.mtls.enabled=true --set values.security.citadelHealthCheck=true > citadel-health-check.yaml
+$ istioctl manifest generate --set values.global.mtls.enabled=true,values.security.citadelHealthCheck=true > citadel-health-check.yaml
 $ kubectl apply -f citadel-health-check.yaml
```

Comment:
Replace with:
istioctl manifest apply --set values.global.mtls.enabled=true,values.security.citadelHealthCheck=true
and remove the kubectl apply line.

Reply:
The yaml file, citadel-health-check.yaml, is used again later in the task so I left it as generate. Maybe the whole task could be improved, but that's true for many of these tasks, and should be done in a separate PR.

Reply:
ok, sounds fine to leave as is.